src/components/Marketplace/MarketplaceScreen.js

```diff
@@ -78,33 +82,50 @@ export default function MarketplaceScreen() {
             produceType,
             palletsAvailable: record.fields['pallets available'] || 0,
             hasAgencyPrice,
+            // fields for sorting
+            'sell by date': record.fields['sell by date'] || 0,
+            'first available date': record.fields['first available date'],
```

Comment:
Is there a reason why the 'sell by date' field is ORed with 0, but the other fields aren't?

Reply:
Nope, I forgot to add null checks for the rest. Fixed it!